manifest.php

```diff
@@ -1,4 +1,5 @@
 <?php
+
```

Comment:
is this line necessary ? 
maybe we can take a look at this config that your using

Reply:
I'm using [php-cs](https://github.com/squizlabs/PHP_CodeSniffer) with default parameters.
After check, it appears that [our jenkins CI](http://qa.kitchen.it.taocloud.org/job/deploy-test-package/job/product-ci%252Ftaoce/lastStableBuild/php-code-sniffer/folder.1867278147/category.-829123767/source.12fb8437-6324-4e4c-b751-bfa13ada634e/#1) is throwing the same error

Reply:
PSR-2 says `blank_line_after_php`: no.
https://www.php-fig.org/psr/psr-2/ search for `blank_line_after_php:`

Reply:
As discussed IRL, it is psr12 interpretation